scripts/disabled_tests/README.md

```diff
@@ -26,7 +26,31 @@ Python 3.8+
 pip install -r requirements.txt
 ```
 
+### Running unittests
+
+#### All tests
+```shell
+# in scripts/disabled_tests
+python -m unittest discover tests
+```
+
+#### Individual tests
+```shell
+# in scripts/disabled_tests
+python -m unittest tests\test_playlist_parser.py
+```
+
```

Comment:
Just tried the test, got the following error:
```
E
======================================================================
ERROR: teststest_playlist_parser (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: teststest_playlist_parser
Traceback (most recent call last):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'teststest_playlist_parser'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)

Reply:
Good thing Windows accept both forward and back slashes ;)

Running this should work fine now:
```shell
# in scripts/disabled_tests
python -m unittest tests/test_playlist_parser.py
```

Tested on Ubuntu 20.04 and Windows. Updated the README in 541e1637141cf34b936013ed662aa6188a2360e8

Reply:
Thanks!